exercises/chapter10/test/Examples.js

```diff
@@ -1,31 +1,32 @@
 "use strict";
```

Comment:
Oo... I should have mentioned this earlier, but we don't need `"use strict";` anymore in ES modules because it's strict by default. That can be fixed in a separate PR thought so that we unblock this one.